starboard/android/shared/audio_track_audio_sink_type.cc

```diff
@@ -433,16 +431,14 @@ SbAudioSink AudioTrackAudioSinkType::Create(
   const SbTime kStartTime = 0;
   // Disable tunnel mode.
   const int kTunnelModeAudioSessionId = -1;
-  // Disable AudioDeviceCallback for WebAudio.
```

Comment:
We should still ensure that audioDeviceCallback is disabled for WebAudio.

Reply:
Fixed.